lib/rgb_matrix/engine.ex

```diff
@@ -50,62 +44,88 @@ defmodule RGBMatrix.Engine do
   end
 
   @doc """
-  Register a `RGBMatrix.Paintable` for the engine to paint pixels to.
-  This function is idempotent.
+  Register a paint function for the engine to send frames to. This function is
+  idempotent.
```

Comment:
It won't ruin it if there's an empty line.  There was an example farther down where it wasn't on its own line, so I wasn't sure of your intent.

As for formatting: yeah I get your point, but I use a plugin in my editor that auto-wraps comments and docstrings at line 80; I can't (easily) put in exceptions, because the formatter will just always re-format them...

Reply:
>There was an example farther down where it wasn't on its own line, so I wasn't sure of your intent.

![image](https://user-images.githubusercontent.com/4755047/86855548-6f558580-c06f-11ea-845c-5ade64268773.png)

Eh, do what you will with it.